From 2c64bff84abde12139cb3b6bf779cf566daeaddc Mon Sep 17 00:00:00 2001
From: Michael Cho <michael@michaelcho.dev>
Date: Thu, 15 Aug 2024 22:49:17 -0400
Subject: [PATCH] Fix build with boost 1.86.0

Boost 1.86.0 changes seem to no longer indirectly include header which
causes build to fail with:
```
uuid-utils.cc:38:58:
error: 'random' is not a class, namespace, or enumeration
```

boost/random/mersenne_twister.hpp has been available since Boost 1.21.2

(cherry picked from commit eed56000b1d68ac083b8e8bea4ff0ea30a1579c4)
---
 pdns/uuid-utils.cc | 1 +
 1 file changed, 1 insertion(+)

Index: pdns-recursor-5.1.1/uuid-utils.cc
===================================================================
--- pdns-recursor-5.1.1.orig/uuid-utils.cc
+++ pdns-recursor-5.1.1/uuid-utils.cc
@@ -30,6 +30,7 @@
 #endif /* BOOST_PENDING_INTEGER_LOG2_HPP */
 #endif /* BOOST_VERSION */
 
+#include <boost/random/mersenne_twister.hpp>
 #include <boost/uuid/uuid_generators.hpp>
 
 // The default of:
